pages/common/qpdf.md

```diff
@@ -11,6 +11,10 @@
 
 `qpdf --empty --pages {{file1.pdf}} {{1,6-8}} --pages {{file2.pdf}} {{3,4,5}} -- {{output.pdf}}`
 
+- Merge all the pages of a list of PDF files and save the result as another one:
```

Comment:
"Another one" might be confusing since you could be talking about lists or pdf files. 
Something like "save the result to a pdf" file would read better IMHO.

Reply:
What about:

    Merge all the pages of a list of PDF files and save the result as a new PDF:

Reply:
Then let us do it for all the descriptions.

Also, this should move one example above, as this is a more general case than the one above.